src/shared/components/dateRangePicker/DatePicker.tsx

```diff
@@ -115,10 +129,12 @@ export default class DatePicker extends PureComponent<Props, State> {
     }
 
     if (inputFormat) {
-      return moment(dateTime).format(inputFormat)
+      const formatter = createDateTimeFormatter(inputFormat, timeZone)
+      return formatter.format(new Date(dateTime))
     }
+    const formatter = createDateTimeFormatter('YYYY-MM-DD HH:mm:ss', timeZone)
```

Comment:
minor nit: let's refer to `DEFAULT_TIME_FORMAT` rather than the string `'YYYY-MM-DD HH:mm:ss'`

Reply:
True, will do that refactor!